fix headers params

rmzaoo opened the pull request.

G4brym reviewed (changes requested):
~~Hey there, thanks for this pr, but for this to work you will also need to update the `extractParameter` function in `parameters.ts` file~~
nevermind, this function is not even used anymore, let me manually test your change

rmzaoo commented:
> ~Hey there, thanks for this pr, but for this to work you will also need to update the `extractParameter` function in `parameters.ts` file~ nevermind, this function is not even used anymore, let me manually test your change

Nice, I had changed it but I've reverted. Thank you 🙏🏻

G4brym commented:
Thanks for this fix, i've opened a new release (v1.0.6) with your change
